react/lesson1.md

```diff
@@ -16,17 +16,72 @@
 
 # React
 
-[React](https://facebook.github.io/react/) is a library by Facebook.
+[React](https://facebook.github.io/react/) is a library by Facebook for building user interfaces. 
+
+Today, the focus is on component based architectures, and React provides a thin layer on top of JavaScript and HTML to allow you to build custom components that can encapsulate behavior and style and can be re-used throughout your application and even exported for others to use.
+
+We already have some existing components in HTML. When you write this:
+
+```html
+<input type="number" name="age" value="5">
+```
+
+..it generates this: 
+
+<img src="assets/html_input.png" width="150">
+
+An "input" box, that only supports numbers, with a set of up/down arrows for changing the value, and a pre-defined value of 5. So the **element tag** tells the browser which element to render, and the attributes specify the settings for that element and any information we want to pass into it.
 
 ## Why React?
-ToDo
+
+Besides providing the tools to build powerful and well encapsulated components, React also takes care of updating the view to match our data.
```

Comment:
Are students familiar with the term "view" in this context. If we can keep the set of vocabulary we use small that will help. Perhaps "rendered component" would be better.

Reply:
Or "display" or even "HTML code" since that's what's happening.

Reply:
Totally agree. I'll try to pay attention to the terms I'm using in the class as well.